modules/sdp/SdpSimulcast.ts

```diff
@@ -0,0 +1,317 @@
+import MediaDirection from '../../service/RTC/MediaDirection';
+import * as MediaType from '../../service/RTC/MediaType';
+
+import * as transform from 'sdp-transform';
+
+const DEFAULT_NUM_OF_LAYERS = 3;
+
+interface Description {
+    type: RTCSdpType;
+    sdp: string;
+}
+
+interface Options {
+    numOfLayers?: number
+}
+
+/**
+ * This class handles SDP munging for enabling simulcast for local video streams in Unified plan. A set of random SSRCs
+ * are generated for the higher layer streams and they are cached for a given mid. The cached SSRCs are then reused on
+ * the subsequent iterations while munging the local description. This class also handles imploding of the simulcast
+ * SSRCs for remote endpoints into the primary FID group in remote description since Jicofo signals all SSRCs relevant
+ * to a given endpoint.
+ */
+export default class SdpSimulcast {
+    private _options: any;
```

Comment:
this should also have the options type or maybe this declaration is not needed (not sure)

Reply:
It is fine but let me push another commit for this.

Reply:
What I've seen is that too many `any` types are making local development slow as this seems to be impacting TypeScript performance, so it's better to avoid that (if you have TS checks enabled in IDE).